metro.config.macos.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * This cli config is needed for development purposes, e.g. for running
+ * integration tests during local development or on CI services.
+ */
+
+const path = require('path');
+const blacklist = require('metro-config/src/defaults/blacklist');
+
+const rnmPath = path.resolve(__dirname, 'node_modules/react-native-macos');
+
+module.exports = {
+  resolver: {
+    extraNodeModules: {
+      'react-native': rnmPath,
+    },
+    platforms: ['macos', 'ios', 'android'],
```

Comment:
Shouldn't this be:
```suggestion
    platforms: ['macos'],
```

Reply:
Technically the react-native-macos package contains macos, ios, and android impls because it is a fork of react-native.   I *do* use the ios impl when testing shared mac/ios source changes.   But the android platform is not useful and I could remove it.